.gitignore

```diff
@@ -0,0 +1 @@
+inode_modules
```

Comment:
this should be node_module you added the "i" accidently hence why your node_modules are being pushed